meson.build

```diff
@@ -273,19 +277,32 @@ if get_option('zsh-completions')
 		'completions/zsh/_sway',
 		'completions/zsh/_swaymsg',
 	)
+	swayfx_zsh_files = [
+		'_swayfx',
+		'_swayfxmsg',
+	]
```

Comment:
Should this be here? Shouldn't `zsh_files` also be removed then?

Reply:
`swayfx_zsh_files` is passed to `install_data()` a couple lines below. It's used for renaming purposes.